src/go/rpk/pkg/config/weak.go

```diff
@@ -533,6 +534,30 @@ func (s *ServerTLS) UnmarshalYAML(n *yaml.Node) error {
 	return nil
 }
 
+func (ss *SeedServer) UnmarshalYAML(n *yaml.Node) error {
+	var internal struct {
+		Address weakString `yaml:"address"`
+		Port    weakInt    `yaml:"port"`
+		// deprecated
+		Host   *SocketAddress `yaml:"host"`
+		NodeID *weakInt       `yaml:"node_id"`
+	}
+	if err := n.Decode(&internal); err != nil {
+		return err
+	}
+	if internal.NodeID != nil {
+		fmt.Println("redpanda yaml: redpanda.seed_servers.node_id is deprecated and unused.")
```

Comment:
nit: this is the only deprecated message that uses a period at the end of the sentence